spec/presenters/finder_presenter_spec.rb

```diff
@@ -68,4 +80,77 @@
       end
     end
   end
+
+  describe "#sort_options" do
+    it "returns an empty array when sort is not present" do
+      expect(presenter.sort_options).to eql([])
+    end
+
+    it "returns sort options when sort is present" do
+      expected_options = "<option value=\"most-viewed\">Most viewed</option>\n<option value=\"updated-newest\">Updated (newest)</option>"
+
+      expect(presenter_with_sort.sort_options).to eql(expected_options)
+    end
+
+    it "returns sort options without relevance when keywords is not present" do
+      expected_options = "<option value=\"most-viewed\">Most viewed</option>\n<option value=\"updated-newest\">Updated (newest)</option>"
```

Comment:
this test is identical to the one starting line 89

Reply:
It's close, but not the same. The test on line 85 uses `presenter`, which has `nil` sort options. This test on line 95 uses `presenter_with_sort`, which has sort options (including relevance).